lib/csv/parser.rb

```diff
@@ -1021,18 +1030,19 @@ def parse_quotable_robust(&block)
           row << value
```

Comment:
Do we need add an element to `quoted_fields` here?

```suggestion
          row << value
          quoted_fields << @quoted_column_value
```

Could you also add a test that uses this method? `CSV.parse(..., strip: true)` will use this method.